frontend/packages/ceph-storage-plugin/src/components/block-pool/body.tsx

```diff
@@ -58,15 +59,8 @@ export const BlockPoolStatus: React.FC<BlockPoolStatusProps> = ({
   );
 };
 
-export type BlockPoolStatusProps = {
-  status: string;
-  name?: string;
-  error?: string;
-  isClusterReady?: boolean;
-};
-
-export const BlockPoolBody: React.FC<BlockPoolBodyProps> = (props) => {
-  const { cephCluster, state, dispatch, isUpdate, errorMessage, showPoolStatus = true } = props;
+export const BlockPoolForm: React.FC<BlockPoolFormProps> = (props) => {
```

Comment:
just copied pool form-related hooks and react elements in to separate FC for easy code understand and maintenance 

old code: 
```
body () {
  code for finding cluster is healthy or not
  code for block pool forms
  return (
    isClusterHealthy: <display form code> ? <block pool status FC>
  )
}
```

new code is
```
blockPoolForm() {
  code for block pool forms
  return(
    <display form code>
  )
}

body (){
   code for finding cluster is healthy or not
   
   return(
      isClusterHealthy: <block pool form FC> ? <block pool status FC>
   )
}

```